src/main/java/core/basesyntax/service/serviceimpl/DataConverterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line at the beginning of the class body.